src/snyk/common/languageServer/languageServer.ts

```diff
@@ -118,7 +119,7 @@ export class LanguageServer implements ILanguageServer {
     };
 
     // Create the language client and start the client.
-    this.client = this.languageClientAdapter.create('Snyk LS', SNYK_LANGUAGE_SERVER_NAME, serverOptions, clientOptions);
+    this.client = this.languageClientAdapter.create('SnykLS', SNYK_LANGUAGE_SERVER_NAME, serverOptions, clientOptions);
```

Comment:
This is necessary if we want to enable trace logs later. The Language Client Id can't have an empty space